src/components/Filter/Filter.js

```diff
@@ -0,0 +1,27 @@
+import cx from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Filter extends React.Component {
```

Comment:
It's stateless now 😄 ...so I think we can prefer that syntax:
https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc

ex:
https://github.com/patternfly/patternfly-react/blob/master/src/components/Wizard/WizardSidebar.js

Reply:
Updated. Thanks.